packages/shared/src/component-utils.ts

```diff
@@ -61,10 +61,14 @@ export const INTERNAL_doFormatting = (config?: ExpandedRouteConfig): string => {
   const key = allowedTypes[0];
   const formattedKey = formattedTypes[0];
 
-  const { maxFileSize, maxFileCount } = config[key]!;
+  const { maxFileSize, maxFileCount, minFileCount } = config[key]!;
 
   if (maxFileCount && maxFileCount > 1) {
-    return `${formattedKey}s up to ${maxFileSize}, max ${maxFileCount}`;
+    if (minFileCount > 1) {
+      return `${formattedKey}s up to ${maxFileSize}, at least ${minFileCount}, max ${maxFileCount}`;
```

Comment:
will this text fit well? maybe

`${minFileCount} - ${maxFileCount} ${formattedKey}s up to ${maxFileSize}`

if it's tight on space. haven't run an example so no idea if there is spaceor not

Reply:
Will check